examples/kepler-integration/src/components/export-video/export-video-panel-container.js

```diff
@@ -161,12 +139,54 @@ export class ExportVideoPanelContainer extends Component {
     });
     // setFileNameDeckAdapter(name.target.value);
   }
-  /* setQuality(resolution){
+  setQuality(resolution) {
     this.setState({
```

Comment:
We should reduce the number of times setState is called. Add an else statement for the "non-match" case and then also set `quality` in each switch.